### ISSUE TITLE
Balanced wight for crossentropyloss

### ISSUE THREAD

**Issue by VladPerish** [yassouali/pytorch_segmentation#40]
How to use the get weights function with the cross entropy loss in multiclass problem?

**Comment by yassouali** [comment]
you can use sklearn [compute_class_weight](https://scikit-learn.org/stable/modules/generated/sklearn.utils.class_weight.compute_class_weight.html). A possible approach is to compute the weights using a subset of the training targets and pass them to the cross entropy loss.

You can try using Focal Loss, that takes into account the unbalanced data using the model's certainty.
